contracts/SimpleQuests.sol

```diff
@@ -0,0 +1,417 @@
+pragma solidity ^0.6.0;
+
+import "@openzeppelin/contracts-upgradeable/proxy/Initializable.sol";
+import "@openzeppelin/contracts-upgradeable/access/AccessControlUpgradeable.sol";
+import "./cryptoblades.sol";
+import "./characters.sol";
+import "./weapons.sol";
+import "./shields.sol";
+import "./Promos.sol";
+import "./util.sol";
+import "./items/Junk.sol";
+import "./items/RaidTrinket.sol";
+import "./SafeRandoms.sol";
+
+contract SimpleQuests is Initializable, AccessControlUpgradeable {
+
+    bytes32 public constant GAME_ADMIN = keccak256("GAME_ADMIN");
+    uint256 public constant SEED_RANDOM_QUEST = uint(keccak256("SEED_RANDOM_QUEST"));
+
+    /* Quest rarities
+    *   Quests templates rarities on (0 - common, 1 - uncommon, 2 - rare, 3 - epic, 4 - legendary)
+    *   Promo quests arrays are accordingly (10 - common, 11 - uncommon, 12 - rare, 13 - epic, 14 - legendary)
+    */
+
+    /* Promo quest templates
+    *   Use setUsePromoQuests(bool usePromoQuests) in order to use promo quest templates, when it's true, it will
+    *   use promo quest templates, when it's false, it will use regular quest templates
+    */
+
+    Characters public characters;
+    Weapons public weapons;
+    Junk public junk;
+    RaidTrinket public trinket;
+    Shields public shields;
+    SafeRandoms public safeRandoms;
+
+    uint8 public constant VAR_COMMON_TIER = 0;
```

Comment:
I know this is some galaxy-brain level shit coming from me, but our patterns so far have avoided using a var index of 0 in other contracts due to it being the default number.
As far as I know there is no case where this could potentially be an issue, and cases are likely to be isolated within the given contracts anyway. But it's food for thought